### PR TITLE
Test on node v6 and latest on Travis

### PR DESCRIPTION
**Pull request by paulkernfeld**

### CODE REVIEW

**Comment by paulkernfeld** [comment]
I'm going to check that this passes Travis on my fork first.

**Comment by paulkernfeld** [comment]
This passes Travis, but I see a lot of deprecation warnings as follows on Node 7:

```
(node:2540) DeprecationWarning: Using Buffer without `new` will soon stop working. Use `new Buffer()`, or preferably `Buffer.from()`, `Buffer.allocUnsafe()` or `Buffer.alloc()` instead.
```

**Comment by paulkernfeld** [comment]
Okay, I think I fixed the deprecation warnings by updating html-tokenize in [this PR](https://github.com/substack/html-tokenize/pull/19).

**Comment by juliangruber** [comment]
> Okay, I think I fixed the deprecation warnings by updating html-tokenize in this PR.

wooot you da man!

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.12"
```
Comment (juliangruber) [inline .travis.yml]: we still want to keep 0.12. i'll merge otherwise :)

Comment (paulkernfeld) [inline .travis.yml]: Okay, I have learned that the buffer-from module exists for this purpose.

**Comment by juliangruber** [comment]
thanks!

**Pull request merged**